dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -180,7 +177,24 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
                 Thread.sleep(TimeUnit.MINUTES.toMillis(1));
             }
         }
-
         return listOfRepos;
     }
+
+    /**
+     * Returns a <code>java.util.Map</code> of GitHub repositories owned by a user. Returned map contains repository names
```

Comment:
Could perhaps say that the Map's key is the name of the repo.

Reply:
Updated PR